htdocs/takepos/invoice.php

```diff
@@ -678,7 +678,10 @@ function fail($message)
 		$printer = new dolReceiptPrinter($db);
 	}
 
-	$headerorder = '<html><br><b>'.$langs->trans('Place').' '.$place.'<br><table width="65%"><thead><tr><th class="left">'.$langs->trans("Label").'</th><th class="right">'.$langs->trans("Qty").'</th></tr></thead><tbody>';
+	$sql = "SELECT label FROM ".MAIN_DB_PREFIX."takepos_floor_tables where rowid=".$place;
+    $resql = $db->query($sql);
+    $row = $db->fetch_array($resql);
```

Comment:
We are working to remove fetch_array in favor of fetch_object (fetch_array return data in a not named array where a change of the order of parameters generate regressions). There is more code to add, but result is more robust and code that use named column instead of field number x is more readable. Can you change this in favor of fetch_object ?